modules/dbfs/multi.go

```diff
@@ -0,0 +1,226 @@
+package dbfs
+
+import (
+	"strconv"
+
+	"fmt"
+
+	"github.com/CodeCollaborate/Server/modules/patching"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+// TODO (wongb): Move this to config
```

Comment:
but I meant it in the sense that there's no reason to expose it to whoever is managing the server